pkg/sql/parser/sql.y

```diff
@@ -16374,6 +16374,7 @@ bare_label_keywords:
 | INVOKER
 | LEAKPROOF
 | PARALLEL
+| QUERY
```

Comment:
we discussed offline, but could you adjust the comment a bit for `bare_label_keywords` and the other rules above and below? i think the information is a bit outdated.

Reply:
I just chatted with @chengxiong-ruan and have it updated. Basically we still want a keyword to be only in one of those "_keyword" lists  (note, `bare_labeled_keywords` is not one of them), but **newly added** keyword should always be added to `bare_labeled_keywords` as well.